src/components/tooltip/Marker.js

```diff
@@ -811,7 +812,7 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
               {/* we don't know the duration if the marker is incomplete */}
               {!marker.incomplete
                 ? marker.dur
-                  ? formatMilliseconds(marker.dur)
+                  ? formatTimestamp(marker.dur)
```

Comment:
Thanks!
I think we should make it slightly less precise by adding some parameters to the call.
What about using `3, 1`? (means 3 significant digits, 1 fractional digit max)

Reply:
Done